pkg/operators/ebpf/types.go

```diff
@@ -33,4 +33,7 @@ const (
 
 	// Prefix used to mark variables used by operators
 	varPrefix = "gadget_var_"
+
+	// Constant defining gadget_l3endpoint_t
```

Comment:
Do we need this comment?

Reply:
We can remove it, 
I just thought since every other prefix had a comment, I'll add one here.
or maybe I can make it more descriptive , something like :
`Constant defining the name of a struct type for filtering IP addresses`